.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   "globals": {
   },
   "rules": {
-    "no-useless-escape": "off"
+    "no-useless-escape": "off",
+    "no-trailing-spaces": "off"
```

Comment:
What introduced this need?

Reply:
It seems that this is failing in travis: https://travis-ci.org/CartoDB/cartodb.js/builds/340449842, but not in my local machine :S